tests/testthat/test_estimate_incubation.R

```diff
@@ -75,6 +93,14 @@ test_that("empirical incubation period distribution matches reference", {
 })
 
 
+test_that("empirical incubation period distribution matches dist reference computed by hand", {
+  skip_on_cran()
```

Comment:
I dont know which tests need `skip_on_cran()`, will let you decide if to leave it in or not.

Reply:
I think this particular test is small enough to let past CRAN :)

IMO: the criteria I have for letting a test on CRAN:

1. The test runs fast (e.g. about a millisecond)
1. If something goes wrong with the test, I can identify what it is relatively quickly
1. I know without a doubt that I control the input and output data